2/contrib/jenkins/fix-permissions

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+# Fix permissions on the given directory to allow group read/write of 
+# regular files and execute of directories.
+find $1 -exec chgrp 0 {} \;
+find $1 -exec chmod g+rw {} \;
+find $1 -type d -exec chmod g+x {} +
```

Comment:
I see ... I was able to use them in jenkins-plugin with no ill-affects (re: point 1)), but yeah I've experienced enough now that 2) resonates (and 2) did not apply in the jenkins-plugin case)

I'll defer.